app/src/test/java/uc/edu/klopfsea/groupscheduler/PlannedEventUnitTest.kt

```diff
@@ -6,48 +6,48 @@ import uc.edu.klopfsea.groupscheduler.Events.PlannedEvent
 
 class PlannedEventUnitTest {
 
-    var name: String = "Dinner"
-    var date: String = "2021-04-26"
-    var startTime: String = "1700"
-    var plannedEvent: PlannedEvent = PlannedEvent(name, date, startTime)
-    lateinit var invalidPlannedEven: PlannedEvent
+    private var NAME: String = "Dinner"
+    private var DATE: String = "2021-04-26"
+    private var START_TIME: String = "1700"
+    private var plannedEvent: PlannedEvent = PlannedEvent(NAME, DATE, START_TIME)
+    private lateinit var invalidPlannedEven: PlannedEvent
 
     @Test
-    fun plannedEvent_maintainsState() {
+    fun `planned event maintains state`() {
```

Comment:
this test and the tests on lines 24, 35, and 46 names were refactored, using backticks and spaces as apposed to double qoutes and uderscores to fit the kotlin conventions described here, https://kotlinlang.org/docs/coding-conventions.html#names-for-test-methods

Reply:
Good 

(but opposed, not apposed.  :) )